app/src/main/java/com/github/se/cyrcle/ui/review/AllReviewsScreen.kt

```diff
@@ -292,6 +297,36 @@ fun AllReviewsScreen(
                                         color = MaterialTheme.colorScheme.onPrimaryContainer,
                                         style = MaterialTheme.typography.bodySmall,
                                         modifier = Modifier.testTag("ReviewText$index"))
+                                    Spacer(modifier = Modifier.height(16.dp))
+
+                                    if (userViewModel.currentUser.value != null) {
+                                      FloatingActionButton(
+                                          onClick = {
+                                            reviewViewModel.selectReview(curReview)
+                                            reviewViewModel.addReport(
+                                                ReviewReport(
+                                                    "TEST0",
+                                                    ReviewReportReason.HARMFUL,
+                                                    userViewModel.currentUser.value?.public?.userId
+                                                        ?: "TESTUSER",
+                                                    curReview.uid),
+                                                userViewModel.currentUser.value!!)
+                                            Toast.makeText(
+                                                    context, "Review reported!", Toast.LENGTH_SHORT)
+                                                .show()
+                                          },
+                                          containerColor = MaterialTheme.colorScheme.error,
+                                          modifier =
+                                              Modifier.align(Alignment.CenterHorizontally)
+                                                  .height(36.dp)
+                                                  .testTag("ReportReviewButton$index")) {
+                                            Text(
+                                                text =
+                                                    "REPORT REVIEW", // Add this string to resources
```

Comment:
Can you please do what the comment says ?

Reply:
Oops, guess I forgot to read my own notes!